### ISSUE TITLE
Make collapse-behavior user-definable per step

### ISSUE THREAD

**Issue by flosell** [flosell/lambdacd#71]
Scenario: You deploy to ci and qa in parallel. The in-parallel parent isn't the interesting part so we might want it to be expanded by default.

Split out of #59 as a nice to have feature but with no clear plan yet how to implement it well.
Could be metadata on the step (like `in-parallel`) but this would require that users set up something like  `expanded-by-default-in-parallel`) which might be a bit annoying. 

Could also be implemented with invisible container steps similar to how `alias` is implemented but then more and more view-logic would leak into the pipeline-structure code, making the pipeline structure harder to read... 

**This issue is nice to have and will probably not be implemented unless there is demand. Give feedback if you need this**

**Comment by flosell** [comment]
This has been here for a while without demand. Closing this for now, feel free to comment to reopen.
